app/src/modules/content/components/version-menu.vue

```diff
@@ -311,23 +330,24 @@ async function onPromoteComplete(deleteOnPromote: boolean) {
 					<div class="grid">
 						<div class="field">
 							<v-input
-								v-model="newVersionKey"
+								v-model="newVersionName"
 								class="full"
-								:placeholder="t('version_key')"
+								:placeholder="t('version_name')"
 								autofocus
-								slug
 								trim
-								:max-length="64"
+								:max-length="255"
 								@keyup.enter="createVersion"
 							/>
 						</div>
+
 						<div class="field">
 							<v-input
-								v-model="newVersionName"
+								v-model="newVersionKey"
 								class="full"
-								:placeholder="t('version_name')"
+								:placeholder="t('version_key')"
+								slug
```

Comment:
In fact, the v-input already has the slug prop, which slugifies the entry when the user inputs a space, though it only seems to fire on an emit event on the input. So might be able to get away with this just emitting something instead of using slugify here? Though I'm not 100% sure.